catalog/core/catalog-core-solr-defaultsolrcollectionprovider/src/main/java/ddf/catalog/provider/solr/defaultsolrcollectionprovider/DefaultSolrCollectionProvider.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.provider.solr.defaultsolrcollectionprovider;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.source.solr.api.impl.AbstractSolrCollectionProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** This is the default collection provider and will always return `catalog_default` */
+public class DefaultSolrCollectionProvider extends AbstractSolrCollectionProvider {
+  static final String DEFAULT_COLLECTION = "default";
```

Comment:
ye i'll go with the former option. not including the suffix, the collection name will then collide with the alias name, cause possible confusion. But if someone is to upgrade to this DDF version, this will require data migration, from the existing `catalog` collection to this new catalog_resource collection